jung-samples/src/main/java/edu/uci/ics/jung/samples/SubLayoutDemo.java

```diff
@@ -282,8 +281,8 @@ private void heightConstrain(Component component) {
 
   private LayoutAlgorithm<String, Point2D> getLayoutAlgorithmFor(
       Class<CircleLayoutAlgorithm> layoutClass) throws Exception {
-    Constructor<CircleLayoutAlgorithm> constructor = layoutClass.getConstructor(PointModel.class);
-    return constructor.newInstance(POINT_MODEL);
+    Constructor<CircleLayoutAlgorithm> constructor = layoutClass.getConstructor();
```

Comment:
I'm glad you spotted that.  :)

Moving forward, I think that the right mechanism here is to define an enum for the layouts to be considered, and then have a method that constructs the appropriate instance based on the enum value.  I think I implemented this in one of the demos but hadn't applied it to all of them yet.